src/SprykerEco/Zed/Heidelpay/Business/HeidelpayFacadeInterface.php

```diff
@@ -36,6 +37,50 @@ interface HeidelpayFacadeInterface
      */
     public function postSaveHook(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponseTransfer): CheckoutResponseTransfer;
 
+    /**
+     * {@inheritdoc}
```

Comment:
You should have actual specification in interface

Reply:
{@inheritdoc} replaced with template
Secification:
     - TBD

Actual text description demands some time for investigation and writing. @alex-galych